graphql-codegen-haskell/src/parse/operation.ts

```diff
@@ -113,6 +114,16 @@ class OperationDefinitionParser {
       )
     }
 
+    node.variableDefinitions?.forEach((v) => {
```

Comment:
Instead of doing this here, it would probably be better done in `parseVariableDefinitions`

Reply:
If you'd prefer, you could also write an issue for what you'd like and I can implement it instead

Reply:
Sure, I've opened the issue here https://github.com/LeapYear/graphql-client/issues/58